Sources/NIOSSL/SSLContext.swift

```diff
@@ -118,6 +118,13 @@ public final class NIOSSLContext {
     private let callbackManager: CallbackManagerProtocol?
     private var keyLogManager: KeyLogCallbackManager?
     internal let configuration: TLSConfiguration
+    
+    /// Access cipher suites applied to the context
+    public var cipherSuites: [NIOTLSCipher] {
```

Comment:
This API does not need to be public, and it's a bit duplicative of `stackOfCipherSuites`. I think we should just make `stackOfCipherSuites` a `var` and `internal` and call it `cipherSuites`.

Reply:
Good call.  Resolved in https://github.com/apple/swift-nio-ssl/pull/293/commits/dc9625669fc700a8f1cb8a064ad4c59a84fc6563